### DISCUSSION TITLE
Linux Permissions

### DISCUSSION THREAD

**Discussion by staticgroup** [OpenSurv/OpenSurv#11] [General]
I personally like to manage my config files outside of the command line using winscp and atom/notepad++ but with the permisions design at what I can see this is challenging. I need to : sudo su > copy /home/opensurv/etc/montior#.yml to /home/myuser  > copy with winscp or edit live > rm .yaml in above director > cp from /home/myuser > chwon file back to opensurv user .... 
I was wondering could we recentalize the config files to /etc/opensurv or similar so any user can rw and not break applicaion readabiliy ?
I would like to start manaing my configs with puppet and foreman and that would be a requirment of this. 
If I am missing something obvious (very very likely) please let me know.; 
thank you

**Comment by SvenVD** [comment]
In your workflow you can do `chown youruser: /etc/opensurv/*` after install and edit the files directly in /etc/opensurv/. OpenSurv does not care if the files are owned by another user as long as they are also readable by OpenSurv. Alternatively, for reference, in version 1.1 you can press 'q' and then you are back at the desktop so you can edit with a GUI on the system itself. And lastly you can use ssh and vi or something like vscode with remote ssh to edit the config files.

**Reply by staticgroup** [reply]
I run Ubuntu server but glad to know it's not user dependent for opensurv to run that resolves my issue !! Thank you

**Comment by staticgroup** [comment]
There is no /etc/opensurv it creates a opensurv user and stores config files under that users home dir its not a symlink back to /etc unfortunately

**Comment by SvenVD** [comment]
Can you re-run the installer as that is the one that should create the symlink to /etc/opensurv

**Comment by staticgroup** [comment]
symlink is up now that i changed to 1.2beta1 branch.
